Octokit.Tests/Clients/RepositoryContentsClientTests.cs

```diff
@@ -89,5 +89,135 @@ public class TheGetContentsMethod
                 Assert.Equal(1, contents.Count);
             }
         }
+
+        public class TheCreateFileMethod
+        {
+            [Fact]
+            public void RequestsCorrectUrl()
+            {
+                var connection = Substitute.For<IApiConnection>();
+                var client = new RepositoryContentsClient(connection);
+
+                string expectedUri = "repos/org/repo/contents/path/to/file";
+                client.CreateFile("org", "repo", "path/to/file", new CreateFileRequest("message", "myfilecontents", "mybranch"));
+                
+                connection.Received().Put<RepositoryContentChangeSet>(Arg.Is<Uri>(u => u.ToString() == expectedUri), Arg.Any<object>());
```

Comment:
:+1:
I missed the second test.. 
Let's just blame it on the fact that I am on my phone.  :smile: